intrastat_base/__manifest__.py

```diff
@@ -1,15 +1,16 @@
-# Copyright 2011-2020 Akretion (http://www.akretion.com)
-# Copyright 2018-2020 brain-tec AG (Kumar Aberer <kumar.aberer@braintec-group.com>)
-# Copyright 2009-2020 Noviat (http://www.noviat.com)
+# Copyright 2011-2021 Akretion (http://www.akretion.com)
+# Copyright 2018-2021 brain-tec AG (Kumar Aberer <kumar.aberer@braintec-group.com>)
+# Copyright 2009-2021 Noviat (http://www.noviat.com)
```

Comment:
FTR, if you don't specify the end date you don't have to update them ever again, as they'll be forever valid

```
# Copyright 2011 Akretion (http://www.akretion.com)
```